synfig-studio/src/gui/canvasview.cpp

```diff
@@ -1464,7 +1485,7 @@ CanvasView::init_menus()
 	action_group->add( Gtk::Action::create("export", Gtk::StockID("synfig-export"), _("Export..."), _("Export")),
 		sigc::hide_return(sigc::mem_fun(*get_instance().get(), &Instance::dialog_export))
 	);
-	action_group->add( Gtk::Action::create("save-all", Gtk::StockID("synfig-save_all"), _("Save All"), _("Save all opened documents")),
+	action_group->add( Gtk::Action::create("", Gtk::StockID("synfig-save_all"), _("Save All"), _("Save all opened documents")),
```

Comment:
Why is this action unnamed?

Reply:
My bad ! I removed that mistakenly :( Will correct this in a minute